api/envoy/service/cluster/v2alpha/health_check_event_service.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+
+package envoy.service.cluster.v2alpha;
+
+import "envoy/api/v2/core/base.proto";
+import "envoy/data/core/v2alpha/health_check_event.proto";
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.service.cluster.v2alpha";
+option java_outer_classname = "HealthCheckEventServiceProto";
+option java_multiple_files = true;
+option java_generic_services = true;
+option (udpa.annotations.file_migrate).move_to_package = "envoy.service.cluster.v3";
+
+// [#protodoc-title: gRPC Health Check Event Service]
+
+// [#not-implemented-hide:]
```

Comment:
How does this compare/contrast with HDS? Would be ideal to have a single way to tackle this.

Reply:
I'll put in my 2 cents. In my mind this is different. This is an event reporting stream vs HDS is asking for information to change data plane behavior. So this is closer in nature to the other 3 event reporting services (metrics, logs, outlier) than it does with HDS. And it fits more nicely into the roll up we are describing in https://github.com/envoyproxy/envoy/pull/10407#discussion_r393900557